Fix capitalization (#7089)
braze-inc/braze-docs: _docs/_api/errors.md

|---|---|
| `5XX Internal Server Error` | Retry your request with exponential backoff.|
| `400 Bad Request` | Bad syntax.|
| `400 No Recipients` | There are no external IDs or segment IDs, or no push tokens in the request.|

> | `400 Invalid Campaign ID` | No Messaging API campaign was found for the campaign ID you provided.|

Author: ```suggestion
| `400 Invalid Campaign ID` | No messaging API campaign was found for your provided campaign ID.|
```
